script/bootstrap

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# script/bootstrap: Resolve all dependencies that the application requires to
+#                   run.
+
+set -e
+
+cd "$(dirname "$0")/.."
```

Comment:
should this print out something to acknowledge that this is a placeholder script e.g. "Bootstrap script not yet implemented"